lms/static/sass/shared/_header.scss

```diff
@@ -10,6 +10,19 @@
   width: 100%;
   height: 76px;
 
+  .doc-link, a.doc-link:hover, a.doc-link:visited {
```

Comment:
Put each of these on their own line. You may even nest them per Sass.

```
.doc-link {
    ...
    &:hover,
    &:focus {
        ...
    }
    &:visited {
        ...
    }
}
```

We also try to follow "out to in" when ordering properties. So, positioning, sizing, margins, borders, padding, etc.

I believe we offer a font-size variable as well as a font-weight variable, though I'm not opposed to these.